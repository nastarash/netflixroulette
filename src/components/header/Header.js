import React from 'react';

function Header() {
  return (
    <header>
      <h1>netflixroullete</h1>
    </header>
  );
}
export default Header;
