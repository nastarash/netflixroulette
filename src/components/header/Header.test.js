import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

it('render correctly', () => {
  const AppComponent = renderer.create(<Header />).toJSON();
  expect(AppComponent).toMatchSnapshot();
});
