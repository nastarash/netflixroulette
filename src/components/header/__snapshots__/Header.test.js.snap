// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<header>
  <h1>
    netflixroullete
  </h1>
</header>
`;
